packages/cache/src/TargetHasher.ts

```diff
@@ -9,6 +11,15 @@ export interface TargetHasherOptions {
   cliArgs?: string[];
 }
 
+function sortObject<T>(unordered: Record<string, T>): Record<string, T> {
```

Comment:
this function is also defined in salt.ts

Reply:
You're right!

Reply:
got it!